apis/refs/v1beta1/sqlinstanceref.go

```diff
@@ -22,3 +34,92 @@ type SQLInstanceRef struct {
 	/* The `namespace` field of a `SQLInstance` resource. */
 	Namespace string `json:"namespace,omitempty"`
 }
+
+type SQLInstance struct {
+	ProjectID       string
+	Location        string
+	SQLInstanceName string
+	ConnectionName  string
```

Comment:
Normally the Reference struct only holds the fundamental identifies (parent + instance ID). 

Does connection always in the form of `${ProjectID}:${Location}:${SQLInstanceName}`? if so, we can build the ConnectionName from the other three and don't need this field.

Reply:
Yeah, makes sense. Even though the form would change in future, I think it still can be built based on other fields.